README.md

```diff
@@ -40,148 +40,190 @@ For more information on the reliable web app pattern, see [Overview](https://rev
 
 ## Prerequisites
 
-* [Azure Subscription](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/)
-* [Visual Studio Code](https://code.visualstudio.com/)
-* [Docker](https://www.docker.com/get-started/)
-* [Permissions to register an application](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app)
-* [Dev Containers extension installed in VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension
+We recommend that you use a DevContainer to deploy this application.  The requirements are as follows:
 
-See [Prequisites](prerequisites.md) for more detailed information about our prerequisites.
+- [Azure Subscription](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+- [Visual Studio Code](https://code.visualstudio.com/).
+- [Docker Desktop](https://www.docker.com/get-started/).
+- [Permissions to register an application](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
+- Visual Studio Code [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers).
 
-## Steps to deploy the reference implementation
+If you do not wish to use a DevContainer, please refer to the [prerequisites](prerequisites.md) for detailed information on how to set up your development system to build, run, and deploy the application.
 
-Note - The following deployment has been tested using devcontainers on **macOS** and **Windows with [Ubuntu on WSL](https://ubuntu.com/wsl)**.
+> **WINDOWS**
+> If you are using Windows, you must enable long paths.  Do the following as a local Administrator:
+>
+> - Open the Registry Editor, navigate to `HKLM\SYSTEM\CurrentControlSet\Control\FileSystem` and set DWORD `LongPathsEnabled` to 1.
+> - Run `git config --system core.longpaths true`
 
-**1. Clone the repo**
+## Steps to deploy the reference implementation
 
-Navigate to your desired directory and run the three following commands:
+The short version (assumes you are already logged into Azure through the Azure CLI and Azure Developer CLI, and that a suitable subscription is selected):
 
 ```shell
 git clone https://github.com/Azure/reliable-web-app-pattern-java.git
 cd reliable-web-app-pattern-java
-code .
+azd config set alpha.terraform on
+azd env new eap-javarwa
+azd env set DATABASE_PASSWORD "AV@lidPa33word"
+azd env set APP_ENVIRONMENT prod
+azd env set AZURE_LOCATION westus3
+azd env set AZURE_LOCATION2 eastus
+azd up
 ```
 
-If cloning in Windows, you may need to configure support for long paths in your environment depending on how long the folder path is you are cloning into. From the Registry Editor, navigate to HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem and set the DWORD LongPathsEnabled to 1.
+The following walk-through assumes you are using a DevContainer inside Visual Studio Code and walks you through executin
```

Comment:
I think it should be `Dev Container` as two words - https://learn.microsoft.com/en-us/shows/beginners-series-to-dev-containers/

Reply:
Fixed throughout.